src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -267,31 +289,24 @@ public RequestOptionsBuilder clearStripeAccount() {
       return setStripeAccount(null);
     }
 
-    public String getStripeVersionOverride() {
+    /** For internal use only. */
+    public String _getStripeVersionOverride() {
       return this.stripeVersionOverride;
     }
 
     /**
-     * Do not use this except for in API where JSON response is not fully deserialized into explicit
-     * Stripe classes, but only passed to other clients as raw data -- essentially making request on
-     * behalf of others with their API version. One example is in {@link
-     * com.stripe.model.EphemeralKey#create(Map, RequestOptions)}. Setting this value in a typical
-     * scenario will result in deserialization error as the model classes have schema according to
-     * the pinned {@link Stripe#API_VERSION} and not the {@code stripeVersionOverride}
+     * This is for internal use only. See {@link com.stripe.model.EphemeralKey#create(Map,
+     * RequestOptions)}. Setting this yourself will result in a version mismatch between your
+     * request and this library's types, which can result in missing data and deserialization
+     * errors.
      *
-     * @param stripeVersionOverride stripe version override which belongs to the client to make
-     *     request on behalf of.
      * @return option builder
      */
-    public RequestOptionsBuilder setStripeVersionOverride(String stripeVersionOverride) {
+    public RequestOptionsBuilder _setStripeVersionOverride(String stripeVersionOverride) {
```

Comment:
can we make this a static method so it doesn't show up in IDE code completion. And maybe name it `unsafeSetStripeVersionOverride`?